docs/non-normative/how-to-write-conventions/t-shaped-signals.md

```diff
@@ -0,0 +1,58 @@
+# T-shaped Signals
+
+Semantic Conventions approaches observability with a T-Shaped mindset. This
+splits signals into two types of a "T" shape:
+
+```mermaid
+block-beta
+        columns 3
+          Broad["Broadly Applicable Signals"]:3
+        columns 3
+          space:1
+          Deep("Rich,\nDeep\nSignals")
+          space:1
```

Comment:
idk if it's possible, but would be cool if this part could be taller, to make the diagram look more like a T

Reply:
I wasn't able to find anything, the block diagram is still beta, so we can revisit this.